src/status_im/utils/ethereum/core.cljs

```diff
@@ -124,8 +125,16 @@
 (defn- sig->method-id [signature]
   (apply str (take 10 (sha3 signature))))
 
-(defn call [web3 params cb]
-  (.call (.-eth web3) (clj->js params) cb))
+(defn call [params callback]
+  (status/call-private-rpc
+   (.stringify js/JSON (clj->js {:jsonprc "2.0"
+                                 :id      1
+                                 :method  "eth_call"
+                                 :params  [params "latest"]}))
+   (fn [response]
+     (if (= "" response)
+       (log/warn :web3-response-error)
+       (callback (get (js->clj (.parse js/JSON response)) "result"))))))
```

Comment:
we have `types/json->clj`, or you don't need to keywordize it here?

Reply:
no I don't need to because I use only result here